docs/dev/legacy/engine_code_overview.md

```diff
@@ -143,8 +147,8 @@ the game goes through in the gamePlay tag.
 
 
 
-```
-<delegate name="battle&quot" javaClass="BattleDelegate"/>
+```xml
+ <delegate name="battle" javaClass="BattleDelegate"/>
```

Comment:
Interesting, looks like I found a bug in the suggest change feature:
In the diff preview this space at the start of the line wasn't visible, but was there.